apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -53,6 +66,15 @@ const App = (): JSX.Element => {
 			<div>
 				<RouterOutlet />
 			</div>
+
+			<button onClick={openModal}>Open Modal</button>
+
+			{isModalOpen && (
+				<Modal onClose={closeModal} open={isModalOpen} title="Test Modal">
```

Comment:
If the state will be stored in `useModal` hook, then for `onClose` we will use `close` function, which will returned by `useModal`, like `const [isOpened, showModal, closeModal] = useModal()`. Also there will be no need in `open` property.